javalib/src/main/scala/java/math/BigDecimal.scala

```diff
@@ -76,6 +79,9 @@ object BigDecimal {
 
   final val ONE = new BigDecimal(1, 0)
 
+  // Since: 19
```

Comment:
We do produce Scaladoc for javalib as a way to see what is implemented. Using a proper doc comments plus an `@since` would make it public. Ditto for the _Since: 9_ below. Not sure if that is helpful to users but it may be.

Reply:
Et voilà. Thank you for the suggestion.  

When does the scaladoc actually get built?  I think building it got
moved out of the "every PR" a while back.  Guess I can check the 
SN GitHub landing page tomorrow.  The scaladoc badges there
tend to run green.